source/polling-srv-records-for-mongos-discovery/polling-srv-records-for-mongos-discovery.rst

```diff
@@ -105,11 +105,15 @@ discovery.  Periodic scan MUST follow these rules:
 - For all verified host names, as returned through the DNS SRV query, the
   driver:
 
-  - MUST add each valid new host to the topology as Unknown
   - MUST remove all hosts that are part of the topology, but are no longer
     in the returned set of valid hosts
   - MUST NOT remove all hosts, and then re-add the ones that were returned.
     Hosts that have not changed, MUST be left alone and unchanged.
+  - If srvMaxHosts_ is zero or greater than or equal to the number of valid
+    hosts, each valid new host MUST be added to the topology as Unknown.
+  - If srvMaxHosts_ is greater than zero and less than the number of valid
+    hosts, valid new hosts MUST be randomly selected and added to the topology
```

Comment:
I suppose you'll want to document the random selection algorithm here as well.

Reply:
This is now done.